manifests/lvs/virtual_server.pp

```diff
@@ -116,6 +124,8 @@
   Optional[Integer[1]] $persistence_timeout = undef,
   Enum['TCP','UDP'] $protocol = 'TCP',
   Optional[Integer[1]] $quorum = undef,
+  Optional[String[1]] $quorum_up = undef,
```

Comment:
are those always paths to files? If so, you could use the `Stdlib::Absolutepath` datatype.

Reply:
As I did not what kind of type to choose, so I selected the one used in [script.pp](https://github.com/voxpupuli/puppet-keepalived/blob/master/manifests/vrrp/script.pp). Is it correct?

Reply:
extract from man `keepalived.conf` : 
> Script to launch when quorum is gained. quorum_up

So data type `Stdlib::Absolutepath` looks correct.

Reply:
> are those always paths to files?

Optionally followed by arguments...